src/_boot.js

```diff
@@ -55,7 +55,9 @@ if (!fs.existsSync(settingsFile)) {
         shell: (process.platform === "win32") ? "powershell.exe" : "bash",
         cwd: electron.app.getPath("userData"),
         keyboard: "en-US",
-        theme: "tron"
+        theme: "tron",
+        lang: "en",
+        pingAddr: "1.1.1.1"
```

Comment:
This is what i would consider an extra/bonus setting; those are not included in the default settings file by default to not bloat the file. (They're mentioned in the wiki)